Makefile

```diff
@@ -199,7 +199,7 @@ pao-functests: cluster-label-worker-cnf pao-functests-only
 pao-functests-only:
 	@echo "Cluster Version"
 	hack/show-cluster-version.sh
-	hack/run-functests.sh
+	hack/run-test.sh -t "test/e2e/performanceprofile/functests" -p "--v -r --fail-fast --skip-package='5_latency_testing,2_performance_update' --flake-attempts=2" -m "Running Functional Tests" -r "--junit-report=report.xml"
```

Comment:
please remember to revert this test back to test `1_performance `instead of `2_performance_update`

Reply:
Sorry but I think I do not understand.

This command line reproduce the same test suite as in `run-functests.sh`

Reply:
@jlojosnegros  what yanir mentioned was we can remove 2_performance_update here as we are anyhow running it in functests-update-only

Reply:
Sorry I read this wrong,  I was looking at the `--skip-package`  which includes the right packages to skip. 
So we should be good here

Reply:
Can you rebase the code, we merged the [PR 663](https://github.com/openshift/cluster-node-tuning-operator/pull/663) which fixes the irq balance test case failure

Reply:
I have already rebased the code. 
Now it is failing due to the timeout as usual https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-node-tuning-operator/631/pull-ci-openshift-cluster-node-tuning-operator-master-e2e-gcp-pao/1663892436478332928#1:build-log.txt%3A1522